mittab/libs/tab_logic.py

```diff
@@ -619,13 +638,13 @@ def avg_deb_speaks(debater):
         return float(sum(real_speaks)) / float(len(real_speaks))
 
 
-def debater_forfeit_speaks(debater):
-    """ Calculate a debater's speaks for a forfeit round
+def debater_forfeit_speaks(debater, round_number):
+    #Calculate a debater's speaks for a forfeit round
 
-    Note that right now we just return 0, but we may want to add support
-    for returning average speaks or some such
-    """
-    return 0.0
+    if (TabSettings.get('lenient_late') >= round_number):
```

Comment:
I'm curious about this implementation choice. My understanding of how most teams want to do avg speaks are when tab directors decide that a particular NoShow team deserves average speaks for a round they should get it. With this implementation, if I understand it correctly, we basically pick a round cutoff where if someone is noshow before that we've ok with it.

I'm curious if this is the right way to do it. On the one hand I like that a tab director can't be corrupt or something and selectively give good speaks to the teams they want, but on the other hand I think tab observers are kind of there specifically to prevent that kind of issue. On the other hand I can understand a situation where a tournament might want to be lenient for a team that traveled a really far way or something, but meh.

What are your thoughts?